src/pad/README.md

```diff
@@ -1,21 +1,282 @@
+place_io_fill -row IO_EAST s8iom0s8_com_bus_slice_10um s8iom0s8_com_bus_slice_5um s8iom0s8_com_bus_slice_1um
+```
+
+### Connecting Ring Signals
+
+Once the ring is complete, use the following command to connect the ring signals
+```
+connect_by_abutment
+```
+
+### Placing Wirebond Pads
+
+To place the wirebond pads over the IO cells
+```
+place_bondpad -bond master
+              [-offset {x y}] \
+              [-rotation rotation] \
+              io_instances
+```
+
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-bond` | Name of the bondpad master |
+| `-offset` | Offset to place the bondpad at with respect to the io instance |
+| `-rotation` | Rotation of the bondpad |
+| `io_instances` | Names of the instances to add bond pads to. |
+
+#### Examples
+```
+place_bondpad -bond PAD IO_*
+```
+
+### Defining a Bump Array
+
+To define a bump array
+```
+make_io_bump_array -bump master \
+                   -origin {x y} \
+                   -rows rows \
+                   -columns columns \
+                   -pitch {x y} \
+                   [-prefix prefix]
+```
+
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-bump` | Name of the bump master |
+| `-origin` | Origin of the array |
+| `-rows` | Number of rows to create |
+| `-columns` | Number of columns to create |
+| `-pitch` | Pitch of the array |
+| `-prefix` | Name prefix for the bump array (default: BUMP_) |
+
+#### Examples
+```
+make_io_bump_array -bump BUMP -origin "200 200" -rows 14 -columns 14 -pitch "200 200"
+```
+
+### Removing Entire Bump Array
+
+To remove a bump array
+```
+remove_io_bump_array -bump master
+```
+
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-bump` | Name of the bump master |
+
+#### Examples
+```
+remove_io_bump_array -bump BUMP
+```
+
+### Removing a Single Bump Instance
+
+To remove a single bump instance
+```
+remove_io_bump instance_name
+```
+
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `instance_name` | Name of the bump |
+
+### Assigning a Net to a Bump
+
+To assign a net to a bump
+```
+assign_io_bump -net net \
+               instance
+```
+
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-net` | Net to connect to |
+| `instance` | Name of the bump |
+
+#### Examples
+```
+assign_io_bump -net p_ddr_addr_9_o BUMP_6_0
+assign_io_bump -net p_ddr_addr_8_o BUMP_6_2
+assign_io_bump -net DVSS BUMP_6_4
+assign_io_bump -net DVDD BUMP_7_3
+assign_io_bump -net p_ddr_addr_7_o BUMP_7_1
+assign_io_bump -net p_ddr_addr_6_o BUMP_7_0
+```
+
+### RDL Routing
+
+To route the RDL for the bump arrays
+```
+route_rdl -layer layer \
+          [-via via] \
+          [-width width] \
+          [-spacing spacing] \
+          [-allow45] \
+          nets
+```
+
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-layer` | Layer to route on |
+| `-via` | Via to use to access terminals on routed net |
```

Comment:
Not clear to me if this is supposed to be for via up to bump or via down to IO pad (or both?). Some PDKs don't have the IO cell PAD pin on the RDL layer.

Reply:
It's for either end, I could provide a separate flag to specify vias for the bump or IO pad, incase you you are not routing on the layer with either side. I'll split it into two.